doc/ocpp_201_status.md

```diff
@@ -1157,10 +1157,10 @@ This document contains the status of which OCPP 2.0.1 numbered requirements have
 | K01.FR.16 |  |  |
 | K01.FR.17 |  |  |
 | K01.FR.19 |  |  |
-| K01.FR.20 | :white_check_mark: |  |
+| K01.FR.20 | :white_check_mark: | This FR hints that `ACPhaseSwitchingSupported` should be per EVSE, but this makes no sense with the rest of the spec. |
```

Comment:
I think this comment highlights the outcome of a conversation, but I don't think incoming contributors would need to know the context of the discussion in order to understand this outcome.

Reply:
I disagree. "makes no sense with the rest of the spec" is not an detailed outcome. **Why** does it make no sense? I am not asking for much, just a link to the conversation so that conscientious contributors **can** read the context if they choose to do so. If you still want to push back on this suggestion, I would like some backup for your thoughts 😄